src/cmd_all/src/playground.rs

```diff
@@ -12,36 +12,110 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::env;
 use std::ffi::OsString;
-use std::net::SocketAddr;
-use std::path::PathBuf;
-use std::process::Command;
-use std::str::FromStr;
+use std::io::Write;
+use std::sync::LazyLock;
 
-use anyhow::{anyhow, Result};
+use anyhow::Result;
 use clap::StructOpt;
-use risedev::{
-    CompactorService, ComputeNodeService, ConfigExpander, FrontendService, HummockInMemoryStrategy,
-    MetaNodeService, ServiceConfig,
-};
 use risingwave_common::config::load_config;
+use tempfile::TempPath;
 use tokio::signal;
 
-fn load_risedev_config(profile: &str) -> Result<(Option<String>, Vec<ServiceConfig>)> {
-    let (config_path, risedev_config) = ConfigExpander::expand(".", profile)?;
-    let services = ConfigExpander::deserialize(&risedev_config)?;
-
-    Ok((config_path, services))
-}
-
 pub enum RisingWaveService {
     Compute(Vec<OsString>),
     Meta(Vec<OsString>),
     Frontend(Vec<OsString>),
     Compactor(Vec<OsString>),
 }
 
+impl RisingWaveService {
+    /// Extend additional arguments to the service.
+    fn extend_args(&mut self, args: &[&str]) {
+        match self {
+            RisingWaveService::Compute(args0)
+            | RisingWaveService::Meta(args0)
+            | RisingWaveService::Frontend(args0)
+            | RisingWaveService::Compactor(args0) => args0.extend(args.iter().map(|s| s.into())),
+        }
+    }
+}
+
+const IDLE_EXIT_SECONDS: u64 = 1800;
+
+/// Embed the config file and create a temporary file at runtime.
+static CONFIG_PATH_WITH_IDLE_EXIT: LazyLock<TempPath> = LazyLock::new(|| {
+    let mut file = tempfile::NamedTempFile::new().expect("failed to create temp config file");
+    write!(
+        file,
+        "[meta]
+disable_recovery = true
+dangerous_max_idle_secs = {IDLE_EXIT_SECONDS}
+max_heartbeat_interval_secs = 600",
```

Comment:
What about making this a separate file and `include_bytes`?

Reply:
What's the benefit? 🤪

Reply:
Highlighting, and easy to maintain. 😄 We can filter the project with `.toml` files if to modify something.

Reply:
Oops. You make idle secs a Rust constant. 😰